pkg/api/describe/describer.go

```diff
@@ -97,7 +98,8 @@ func describeBuilderImage(config *api.Config, image string, out io.Writer) {
 		Tag:                config.Tag,
 		IncrementalAuthentication: config.IncrementalAuthentication,
 	}
-	build.GenerateConfigFromLabels(c)
+	pr, _ := docker.GetBuilderImage(c)
```

Comment:
but now pr could be nil, in which case GenerateConfigFromLabels is going to error anyway, seems better to report the error as early as possible, otherwise it's going to get wrapped/lost (before the error from the pull got bubbled up as is)